telco-hub/LSO/Subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+spec:
+  channel: stable
+  installPlanApproval: Automatic 
+  name: local-storage-operator
+  source: redhat-operators
```

Comment:
The hub cluster will operate in a disconnected environment with custom catalog(s).